tgui/packages/tgui/interfaces/Hypertorus/Recipes.tsx

```diff
@@ -110,25 +121,23 @@ const MemoRow = (props) => {
   );
 };
 
-MemoRow.defaultHooks = activeChange;
-
-const GasCellItem = (props) => {
+const GasCellItem = (props: GasCellProps) => {
   const { gasid, ...rest } = props;
+
+  if (!gasid) return <Table.Cell />;
```

Comment:
this fixes the second error. this just fell through before, there would be no id so the table and inner box would return blank table cell. i don't want getGasLabel to get called without a string at all, so denied

Reply:
👍